### DISCUSSION TITLE
初期セットアップが上手くいきません

### DISCUSSION THREAD

**Discussion by okdhryk** [pf-robotics/kachaka-api#56] [Q&A] [answered]
APIについてのディスカッションでは無いのですが、iPhoneアプリからのセットアップが上手くいきません。
下記の手順でセットアップを実施しましたが最後まで行けません。

iPhoneのアプリを起動すると、セットアップ画面が表示されるので、手順に従って進みます。
「よろしく」→「準備OK」→「OK」→「設置OK」→「設置OK」→「ONにした」→「接続OK」→「わかった」
SSIDとWi-Fiパスワードを入力する
「次へ」→カチャカが起動したら次へ→「次へ」
QRコードが表示されるので、カチャカのフロントカメラにかざしても何も変化なしです。

？Wi-Fi設定できない場合に書いてある通り、
「ねぇカチャカ、ワイファイの設定」と話しかけると
「ワイファイの設定を開始します・・・・・見せて下さい」
と発話するので、先ほどのQRコードを見せると
「ワイファイの設定ではなく、初期設定のコードが表示されました」
「スマートフォンアプリからワイファイ再設定を選択して下さい」
と言われます。

スマートフォンアプリにWi-Fi再設定の項目がありません。
スマートフォンアプリの起動画面（セットアップ）の右上の歯車をクリックすると、
アカウントとヘルプ・お問合せ、設定・情報のメニューでカチャカは登録されていません。

一旦、本体を工場出荷状態にする（方法は知りませんが...）のがいいのでしょうか？

**Comment by nozaki-pfr** [comment]
カチャカを利用いただきありがとうございます

>「ねぇカチャカ、ワイファイの設定」と話しかけると
「ワイファイの設定を開始します・・・・・見せて下さい」

ここから推察するに、カチャカはすでに別の端末などからセットアップ済みでしょうか？
その場合、セットアップした端末では右下の歯車ボタンでカチャカが表示されますでしょうか？

歯車ボタンで表示されるカチャカの一覧は、接続先のWiFiによらず表示されるはずなのでこのような質問をしております。

お手数おかけします。よろしくお願いいたします。

**Comment by okdhryk** [comment]
先ほど、Facebookで寺田さんに色々教えていただきました。

＞ここから推察するに、カチャカはすでに別の端末などからセットアップ済みでしょうか？
大学の研究室で動かしていたカチャカを自宅に持ってきました。

＞その場合、セットアップした端末では右下の歯車ボタンでカチャカが表示されますでしょうか？
同じiPhoneのアプリですが表示されません。
アプリを起動すると初期設定が自動的に始まるのですが、最後にQRを見せるとそれ以上進みません。

現状ですが、Wi-Fiの設定はiPhoneのショートカットで作ったQRコードを見せることで、自宅のWi-Fiにつながっています。
iPhoneも同じW i-Fiに繋がっているのですが、アプリを立ち上げてもカチャカは見えず、初期設定が動くだけです。


--
岡田浩之
玉川大学 脳科学研究所
先端知能・ロボット研究センター・主任教授
（NPO）ロボカップ日本委員会・会長
URL :*okadanet.notion.site*

2023年9月16日(土) 16:36 nozaki-pfr ***@***.***>:

**Comment by okdhryk** [comment, marked as answer]
その後ですが、初期設定のQRコードを後ろのカメラに見せることでうまく行きました。
なぜ前のカメラだとダメだったのかは謎ですが、今のところうまくいってます。

**Comment by nozaki-pfr** [comment]
おそらくteradaとやりとりしていると思いますが、こちらでも共有いただきありがとうございます！
